test/QuestionAPI-test.js

```diff
@@ -1,14 +1,14 @@
 const { expect } = require("chai");
-const { Certificate } = require("crypto");
-const { utils } = require("ethers");
 const { ethers } = require("hardhat");
 const { mineBlocks, add, BN } = require("./utils");
 
-describe("Question Factory Contract", function () {
-  let questionFactory;
+describe.only("Question API Contract", function () {
```

Comment:
can we remove .only?

Reply:
whoops yes I'll push that up shortly, and before merging